test/model.test.js

```diff
@@ -897,7 +897,7 @@ describe.onServer('Remote Methods', function() {
       request(app).get('/TestModels/saveOptions')
         .expect(204, function(err) {
           if (err) return done(err);
-          expect(actualOptions).to.eql({accessToken: null});
```

Comment:
so... that's funny : you asked 29 days ago why this change was required, and i could not remember why. It happens that the test as-is fails since actualOptions is no longer equal to `{accessToken: null}`but to `{accessToken: null, authorizedRoles: {...}}`.
So i reverted my fix, and then you had the error you mentioned in [your comment here](https://github.com/strongloop/loopback/pull/2957#pullrequestreview-25762621)

i rolled the fix back, testing if accessToken is null is actually what we are testing here